src/env/setup_vm.sh

```diff
@@ -1,48 +1,99 @@
-# author: Pavel Yadlouski
```

Comment:
No reason for setup_vm to be part of the library

Reply:
agree